tests/langchain/test_langchain_model_export.py

```diff
@@ -1020,6 +1007,9 @@ def load_db(persist_dir):
     version.parse(langchain.__version__) in (version.parse("0.1.14"), version.parse("0.1.15")),
     reason="LangChain 0.1.14 and 0.1.15 has a bug in loading SQLDatabaseChain",
 )
+@pytest.mark.skipif(
+    IS_LANGCHAIN_03, reason="Saving SQLDatabaseChain does not work with LangChain 0.3.0"
```

Comment:
note: Similar to `QAEvalChain`, the `SQLDatabaseChain` is not officially deprecated. But this has been staying in experimental package for a year so most likely not high priority.

Reply:
Mostly subsumed by agent tool calling functionality. Fine to deprioritize